Library/Formula/aescrypt-packetizer.rb

```diff
@@ -62,10 +60,17 @@ def caveats; <<-EOS.undent
     original_contents = "What grows when it eats, but dies when it drinks?"
     path.write original_contents
 
-    system bin/"paescrypt", "-e", "-p", "fire", path
+    # make sure that the right binary name is used
+    if build.with? "default-names"
+      name = "aescrypt"
+    else
+      name = "paescrypt"
+    end
+
+    system bin/name, "-e", "-p", "fire", path
     assert File.exist?("#{path}.aes")
 
-    system bin/"paescrypt", "-d", "-p", "fire", "#{path}.aes"
+    system bin/name, "-d", "-p", "fire", "#{path}.aes"
```

Comment:
We don't normally bother trying to test for the situation when the user has chosen to override a Homebrew default, which is why I didn't bother doing so here before. Jack commented earlier, he'll know one way or another.